spec/policy_machine_storage_adapters/active_record_spec.rb

```diff
@@ -311,6 +311,93 @@
         end
       end
 
+      describe 'accessible_objects_for_operations' do
+        context 'direct only' do
+          context 'when there are directly accessible objects' do
+            before do
+              priv_pm.add_association(color_1, creator, object_6)
+              priv_pm.add_association(color_2, creator, object_7)
+            end
+
+            it 'returns objects accessible via multiple operations' do
+              result = priv_pm.accessible_objects_for_operations(
+                user_1,
+                [create, paint],
+                direct_only: true
+              )
+              # expected:
+              # {
+              #   create.to_s => [object_6.stored_pe, object_7.stored_pe],
+              #   paint.to_s => [object_6.stored_pe, object_7.stored_pe],
+              # }
+              expect(result.keys).to contain_exactly(create.to_s, paint.to_s)
+              expect(result[create.to_s]).to contain_exactly(object_6.stored_pe, object_7.stored_pe)
+              expect(result[paint.to_s]).to contain_exactly(object_6.stored_pe, object_7.stored_pe)
+            end
+
+            it 'works with filters' do
+              filters = { user_attributes: { color: color_1.color } }
```

Comment:
Neutral question: Should `filters` and possibly other parameters such as `operations` be initialized by `let` blocks?

Reply:
I was thinking the same thing for the array. One positive is it prevents unintentional values but the negative is needing to scroll a bit to reference the values.